test/compress/collapse_vars.js

```diff
@@ -1152,60 +1256,3 @@ collapse_vars_short_circuited_conditions: {
         function i6(x) { var a = foo(), b = bar(); if (baz()) return b; else return a; }
     }
 }
-
-collapse_vars_regexp: {
```

Comment:
This test was added originally as result of a bug fix. Just because the implementation with respect to `is_const` changed does not mean the test should be dropped.